packages/design-system/src/composables/useIsMobile/index.ts

```diff
@@ -0,0 +1,14 @@
+import { computed } from 'vue'
+import { useWindowSize } from '@vueuse/core'
+
+export const useIsMobile = () => {
+  const { width } = useWindowSize()
+
+  const isMobile = computed(() => {
+    return width.value < 768
```

Comment:
Unsure if `768` is the right number here since we're using `640` as mobile breakpoint in the runtime. I think it makes sense to align that. But maybe we need more refinement of the breakpoints in general.

Reply:
I think for for 640 is perfectly fine 👍